.github/workflows/test.yml

```diff
@@ -72,6 +72,7 @@ jobs:
   # 2) Test each changed notebook in parallel
   # ---------------------------------------------------------
   test_notebooks:
+    if: ${{ needs.gather_notebooks.outputs.notebooks != '[]' }}
```

Comment:
I saw this error pop up. What's the tldr on this fix?

Reply:
fix is to skip the test step if notebooks array is empty, which will be the case sometimes!